lib/gooddata/lcm/actions/create_segment_masters.rb

```diff
@@ -47,6 +47,8 @@ def call(params)
           domain = client.domain(domain_name) || fail("Invalid domain name specified - #{domain_name}")
           domain_segments = domain.segments
 
+          data_product = params.data_product
```

Comment:
Refers to 'params' more than self (maybe move it to another class?) (FeatureEnvy)